### ISSUE TITLE
Non-translateable story - Smuggler (Gor-Mor individual)

### ISSUE THREAD

**Issue by DukeDukeEm** [DukeDukeEm/Translation_ShortestTripToEarth_German#85]
![image](https://user-images.githubusercontent.com/69652463/116825990-1fadf080-ab92-11eb-932d-b4b63f694789.png)

Smuggler L2 explosives smuggler, gormor.prefab

**Comment by Dimon485** [comment]
but where is the title? o_O

**Comment by DukeDukeEm** [comment]
Right, the title is probably missing too. It's a follow up text so I didn't think about it being missing in the whole.

**Comment by Duke-M-commits** [comment]
seems to have been fixed by the game devs
